### PR TITLE
[CRO-414] Update for use with `tango-and-cash`

### PR DESCRIPTION
**Pull request by jritterbush**
Update the script to be used with our Maxymiser test building library.

- Throttle API calls to avoid rate limiting from MM
- Add additional allowed properties to `elements` methods
- Add create method to `campaign-actions`
- Change out defaults endpoint from 🇪🇺  to 🇺🇸 , because we are of the class of lazy Americans who don't want to override default settings. We want to be the default.
- Use `yarn` to lock down versions better

### CODE REVIEW

**Comment by jritterbush** [comment]
@ericcecchi @ericmikkelsen @conwaydev and this is also a WIP, but much closer to final from what I can tell. I've modified the original a bit to make it align better with our needs and with the latest options the REST API offers.

File: __tests__/campaign-actions-test.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ const request = require('superagent');
 const config = require('../mock/superagent-mock');
 var superagent = require('superagent-mock')(request, config);
 
-var authPath = 'https://api-auth-eu.maxymiser.com/oauth2/v1';
-var basePath = 'https://api-eu.maxymiser.com/v1';
+var authPath = 'https://api-auth-us.maxymiser.com/oauth2/v1';
+var basePath = 'https://api-us.maxymiser.com/v1';
 var credentials = {
```
Comment (ericcecchi) [inline __tests__/campaign-actions-test.js:9]: You could add a test here for the new `create` method.

File: README.md [hunk 2]
```diff
@@ -409,6 +412,42 @@ api.campaigns.actions.get({
...
+> http://docs.oracle.com/cloud/latest/marketingcs_gs/OMCGF/op-sites-%7Bsite-id%7D-sandbox-campaigns-%7Bcampaign-id%7D-actions-post.html
+
+```javascript
+api.campaigns.actions.create({
+  siteName: 'test.com',
+  campaignName: 'a31-test',
+  elementName: 'renderer',
+  // siteId: 'MzIxMzM',
+  // campaignId: 'MDA2MjYx',
+  // elementId: 'MDMyMDU4',
```
Comment (ericcecchi) [inline README.md]: `elementId` and `elementName` aren't actually part of the actions request schema.

**Review by ericcecchi — CHANGES_REQUESTED** [review]

File: __tests__/campaign-actions-test.js (hunk 1 above)
Comment (jritterbush) [reply __tests__/campaign-actions-test.js:9]: done and passing

**Review by ericmikkelsen — APPROVED** [review]

**Review by ericcecchi — APPROVED** [review]

**Pull request merged**